pcdsdevices/epics_motor.py

```diff
@@ -56,15 +56,60 @@ class EpicsMotorInterface(FltMvInterface, EpicsMotor):
     disabled = Cpt(EpicsSignal, ".DISP", kind='omitted')
     # Description is valuable
     description = Cpt(EpicsSignal, '.DESC', kind='normal')
+    # Current Dial position
+    dial_position = Cpt(EpicsSignalRO, '.DRBV', kind='normal')
 
     tab_whitelist = ["set_current_position", "home", "velocity",
-                     "enable", "disable"]
+                     "enable", "disable", "check_limit_switches"]
 
     def __init__(self, *args, **kwargs):
         # Locally defined limit overrides, note can only make limits narrower
         self._limits = (-math.inf, math.inf)
         super().__init__(*args, **kwargs)
 
+    def format_status_info(self, status_info):
+        """
+        Override status info handler to render the motor.
+
+        Display motor status info in the ipython terminal.
+
+        Parameters
+        ----------
+        status_info: dict
+            Nested dictionary. Each level has keys name, kind, and is_device.
+            If is_device is True, subdevice dictionaries may follow. Otherwise,
+            the only other key in the dictionary will be value.
+
+        Returns
+        -------
+        status: str
+            Formatted string with all relevant status information.
+        """
+        lines = []
+
+        try:
+            description = status_info['description']['value']
+            name = f'{description}: {self.prefix}'
+        except KeyError:
+            name = ' '.join(self.prefix.split(':'))
+            name = f'{name}: {self.prefix}'
+        try:
+            units = status_info['user_setpoint']['units']
+            dial = status_info['dial_position']['value']
+            user = status_info['position']
+        except KeyError as err:
+            logger.error('Key error %s', err)
```

Comment:
We probably need to do something even if these fail- as written, the printout will just be the error message, rather than a best-effort partial printout (and then we get a new bug report)

Reply:
Yeah i see, maybe i should just use `.get()` here and will just print `None`  if can't get it?

Reply:
This might be misinforming though - we probably want to know when we could get the info and it is actually `None` vs. when we could not get the info...

Reply:
I think `'N/A'` is a good call as a placeholder- as you say, there is a difference between the result being `None` and the result failing.

Reply:
ok i'll go for that then, hopefully the users will not really see the `N/A` much

Reply:
I agree, it seems like it should be a really rare codepath- only if the internals totally break or if a subclass reimplements something in a strange way?